kolena/dataset/evaluation.py

```diff
@@ -112,14 +112,28 @@ def download_results(
     model: str,
 ) -> Tuple[pd.DataFrame, List[Tuple[EvalConfig, pd.DataFrame]]]:
     """
-    Fetch results given dataset name and model name.
+    Download results given dataset name and model name.
+
+    Concat dataset with results:
+
+    ```python
+    import kolena
+    import pandas as pd
+    from kolena.dataset import download_results
+
+    kolena.initialize(verbose=True)
+
```

Comment:
We should be OK to omit these in this snippet — they're easily inferrable

Reply:
nice call, just removed them